install.sh

```diff
@@ -53,6 +53,44 @@ printSuccess() {
   echo "${green}${msg[*]}${reset}"
 }
 
+checkJwt() {
+  local is_sandbox=$1
+  local php_path=$2
+  local environment="production"
+  if [[ $is_sandbox ]]; then environment="sandbox"; fi
+  if [[ -f pub/index.php ]]
+  then
+    # shellcheck disable=SC2016
+    $php_path -r '
+      require "app/bootstrap.php";
+      \Magento\Framework\App\Bootstrap::create(BP, $_SERVER);
+      try {
+          $result = [];
+          $om = \Magento\Framework\App\ObjectManager::getInstance();
+
+          $environmentFactory = $om->create(\Magento\ServicesConnector\Model\EnvironmentFactory::class);
+          $jwtToken = $om->create(\Magento\ServicesConnector\Api\JwtTokenInterface::class);
+
+          $envObject = $environmentFactory->create("'$environment'");
+          $apiKey = $envObject->getApiKey("swat");
+          $privateKey = $envObject->getPrivateKey("swat");
+          $signature = $jwtToken->getSignature($privateKey);
+          $result["ApiKey"] = $apiKey;
+          $result["Signature"] = $signature;
+          echo json_encode($result);
+          return 0;
+      } catch (Exception $e) {
+        echo $e;
+        return 1;
+      }'
+      exit_code=$?
+      if [[ $exit_code != 0 ]]; then error_exit "php error"; fi
+      printf "\nJWT Token - OK\n"
```

Comment:
Use echo, so you don't need new lines

Reply:
I need because output from script does not have new line.